### DISCUSSION TITLE
Any way to use Micronaut method interceptor work in Springboot (aop alliance)

### DISCUSSION THREAD

**Discussion by subash89** [micronaut-projects/micronaut-core#8972] [General] [answered]
Hi, I am basically trying to write some common AOPs that will run on micronaut as well as could be reused in springboot. 
I know micronaut capable of running spring written constructs using annotation mappers etc. But this question is more like reverse compatibility. Microaut > Springboot.

**Comment by subash89** [comment]
Any update please ? is micronaut support aop alliance constructs i.e method interceptors

**Reply by graemerocher** [reply, marked as answer]
You would have to write some kind of adapter that invokes Micronaut advice. No idea if that is technically feasible or not.
